### ISSUE TITLE
Player stream time can show "undefined" 

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#3857]
### Share your bug report, feature request, or comment.

I'm not sure exactly in what case this happens, but we should be able to easily reproduce it in Storybook by trying different time combinations.

![image](https://github.com/user-attachments/assets/d2f50f2f-ade8-4f3d-a250-2d989b3fc1b7)

**Comment by cnfairall** [comment]
Hi! I would like to take this on :)

**Comment by gabek** [comment]
Thanks so much, @cnfairall!

**Comment by cnfairall** [comment]
@gabek I'm stuck in setting up dev environment. sent you msg on rocket chat as well. I have go installed but I don't think I managed to install anything else, not sure how
![Screenshot (10)](https://github.com/user-attachments/assets/c238a865-bcaf-4e1d-a91d-7a98890f5eff)

**Comment by cnfairall** [comment]
hey, I don't think I'll be able to do this ticket. I tried for over a week to set up my environment but it proved too complicated for my knowledge level. sorry about it!

**Comment by zockicookie** [comment]
@cnfairall I had some problems with the first setup as well. You will need to set up ffmpeg and gcc. I managed to avoid the cgo error by calling "CGO_ENABLED=1 go run main.go" when powering up the backend.

**Comment by zockicookie** [comment]
@gabek since this is open again, I would love to try my luck with this one.

**Comment by gabek** [comment]
Sure, thank you @zockicookie!

**Comment by mahmed2000** [comment]
Does this happen on builds after #3727 was fixed?

**Comment by gabek** [comment]
@mahmed2000 I thought it was, but that might be incorrect. I think it's good to get this test in that @zockicookie is working on, and then we can close this since it hasn't been able to be reproduced, and I could be wrong on the version(s) that I saw this on. Thanks for chiming in.

**Comment by RonMordo** [comment]
Hi, if availble i would like to try to solve the problem.

**Comment by gabek** [comment]
We actually think it's probably fixed, and it might be an outdated version that this was seen in. I'm going to go ahead and close this and if it happens again after `v0.2.0` is released I'll go ahead and reopen it. Thanks for wanting to help, do feel free to chime in on another issue if you'd like!
